connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/InboundEndpointTest.java

```diff
@@ -73,4 +73,30 @@ public void testDataReturnedForWebhookConnectorExecutableSubclass() {
     assertEquals(1, response.size());
     assertEquals("myPath", response.get(0).data().get("path"));
   }
+
+  @Test
+  public void executableClassNullHandledCorrectly() {
+    var executableRegistry = mock(InboundExecutableRegistry.class);
+    when(executableRegistry.query(any()))
+        .thenReturn(
+            List.of(
+                new ActiveExecutableResponse(
+                    UUID.randomUUID(),
+                    null, // executable class is null
+                    List.of(
+                        new InboundConnectorElement(
+                            Map.of("inbound.context", "myPath", "inbound.type", "webhook"),
+                            new StandaloneMessageCorrelationPoint(
```

Comment:
FYI for the 8.6 backport PR (mentioning just in case):

You'll need to edit this to use the 3-arg constructor that's available in the `release/8.6`

Reply:
Thanks for pointing this out!